executor/executable/controllabletask.go

```diff
@@ -387,17 +392,35 @@ func (t *ControllableTask) Kill() error {
 			"targetList": cmd.TargetList,
 		}).
 		Debug("state DONE not reached, about to commit transition")
+		
+		// Call cmd.Commit() asynchronous
+		commitDone := make(chan *CommitResponse)
+    	go func() {
+			var cr CommitResponse
+			cr.newState, cr.transitionError = cmd.Commit()
+			commitDone <- &cr
+		}()
+		
+		// Set timeout cause OCC is locking up so killing is not possible. The following approach
+		// help us to bypass the OCC transition, so we can kill the tasks on force destroy env. 
+		// Currently this will run for every Kill message receive (force or not).
+		// TODO: Find a better way to distinguish force from plain Kill message.
+		var _cr *CommitResponse
```

Comment:
Leading underscore is not really a convention in Go, so it confused me a bit to see it here. Its usefulness is mostly if you want to have a type name with a leading capital, but keep it unexported (package private), e.g. `_MyType` instead of `MyType`, or perhaps for function-local type definitions. I propose `commitResponse` instead.